thirdparty/minizip/CMakeLists.txt

```diff
@@ -13,14 +13,17 @@ assert_var_defined(AR)
 assert_var_defined(LDFLAGS)
```

Comment:
Also, I meant like this. ^_^ (But see @NiLuJe's comment https://github.com/koreader/koreader-base/pull/1198#discussion_r493751534.)
```suggestion
assert_var_defined(CFLAGS)
assert_var_defined(LDFLAGS)
```

Reply:
@JasonInOttawa There's still a standing review comment! ;-)

Reply:
Thank you!